cleanrl/rnd_ppo.py

```diff
@@ -692,14 +692,14 @@ def forward(self, x):
         return self.network(x)
 
     def get_action(self, x, action=None):
-        logits = self.actor(self.forward(x))
+        logits = self.actor(self(x))
         probs = Categorical(logits=logits)
         if action is None:
             action = probs.sample()
         return action, probs.log_prob(action), probs.entropy()
 
     def get_value(self, x):
-        features = self.forward(x)
```

Comment:
RND needs to be completely refactored: see #127. I would revert changes here because we would no longer use a `forward` function in `rnd_ppo.py and instead use a `get_action_and_value` Function.